requirements.txt

```
@@ -1,6 +1,6 @@
-docutils==0.16
+docutils==0.18
```

Actually saw that sphinx_rtd_theme is only compatible up to 0.17 so pulling this back and trying again.
```suggestion
docutils==0.17
```